esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -260,6 +259,68 @@ EspHttpBinding::EspHttpBinding(IPropertyTree* tree, const char *bindname, const
     }
     if(m_challenge_realm.length() == 0)
         m_challenge_realm.append("ESP");
+
+    if(!m_secmgr.get())
+        return;
+
+    processName.set(procname);
+    const char* authDomain = bnd_cfg->queryProp("@authDomain");
+    if (!isEmptyString(authDomain))
+        domainName.set(authDomain);
+    else
+        domainName.set("default");
+    domainSessionSDSPath.setf("%s/%s[@name=\"%s\"]/%s[@name=\"%s\"]/", PathSessionRoot, PathSessionProcess,
+        procname, PathSessionDomain, domainName.str());
+
+    domainAuthType = AuthPerRequestOnly;
+    VStringBuffer xpath("AuthDomains/AuthDomain[@domainName=\"%s\"]", domainName.get());
+    IPropertyTree* authDomainTree = proc_cfg->queryPropTree(xpath);
+    if (!authDomainTree)
```

Comment:
You might want to default to session based, but default configuration values for it.  Otherwise environments will be slow to upgrade the config.  Unless you think its better not to.  Can you think of issues with doing that?  Operations can always turn it off.